### PR TITLE
added spanish translations

### PR DESCRIPTION
**Pull request by crisgarner**
Translations in Spanish #117

### CODE REVIEW

**Review by ChialiT — APPROVED** [review]

File: app/i18n/locales/es/about-page.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "Estamos construyendo una biblioteca de herramientas de desarrollo, artículos de investigación y prototipos que son de código abierto y gratuitos para que todos los usen. Esperamos que nuestros recursos inspiren a las personas a innovar en las tecnologías que sus comunidades necesitan."
+      ]
+    },
+    {
+      "title": "02. Privacidad por defecto",
+      "description": [
+        "Creemos que la privacidad es un derecho fundamental. Queremos ser parte de la construcción de un internet que se aleje de prácticas invasivas de datos y, en cambio, brinde a las personas opciones reales sobre quién tiene acceso a su información personal. La autorización debe ser específica para el propósito, revocable, informada y no coercitiva.",
+        "Creamos herramientas que ayudan a las personas a autenticarse de manera segura, realizar transacciones confidenciales en el blockchain y respetar y preservar la privacidad del usuario."
```
Comment (aguzmant103) [inline app/i18n/locales/es/about-page.json:18]: _blockchain_ en cursiva si usamos el angliciscmo, sino cadena de bloques (suena mejor blockchain ;) )
https://www.rae.es/observatorio-de-palabras/blockchain

File: app/i18n/locales/es/about-page.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      ]
+    },
+    {
+      "title": "03. Escalabilidad para comunidades",
+      "description": [
+        "Las pruebas de conocimiento cero (ZK Proofs) pueden verificar cálculos de manera rápida y económica, ayudando a sistemas descentralizados como Ethereum a volverse más eficientes. Investigamos, diseñamos y compartimos soluciones de escalabilidad que cualquiera puede usar para contribuir a una infraestructura pública digital más sólida y práctica.",
```
Comment (aguzmant103) [inline app/i18n/locales/es/about-page.json:24]: (zero-knowledge proofs)

File: app/i18n/locales/es/common.json [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  "error": {
+    "404": {
+      "title": "404: Página no encontrada.",
+      "description": "La página que estás buscando podría haber sido eliminada, haber cambiado de nombre o estar temporalmente no disponible."
+    }
+  },
+  "tags": {
+    "build": "Construir",
+    "play": "Jugar",
+    "research": "Investigación",
```
Comment (aguzmant103) [inline app/i18n/locales/es/common.json:42]: Investigar

**Review by aguzmant103 — CHANGES_REQUESTED** [review]

**Review by AtHeartEngineer — APPROVED** [review]

File: app/i18n/locales/es/about-page.json (hunk 1 above)
Comment (crisgarner) [reply app/i18n/locales/es/about-page.json:18]: Not sure if we can add cursive in this json, I'll leave it as cadena de bloques 🖖

**Comment by crisgarner** [comment]
Added another commit with @aguzmant103 suggestions

**Review by kalidiagne — APPROVED** [review]

**Review by CPerezz — COMMENTED** [review]
Loving this!!! Thxz!!!

**Pull request merged**